recipes/ligo-scald/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "ligo-scald" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "88b9ff3b881a381dbd62467b37bbc55fd88b6e999e33e1a1b8b6d3c28952a094" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
```

This looks like it could be a noarch package:
```suggestion
  noarch: python
```

---

I added that since one of the dependencies, `python-lal`, doesn't currently support windows. Should I still be using the `noarch: python` option in `build`?

---

Yes, you can use noarch as long as yo7 don't use selectors in your dependencies, it's just that when someone tries to install the package on windows they will be told that they can't.

---

Thanks all, I updated `meta.yaml` to reflect this.

---

@chrisburr, @duncanmmacleod: switching to noarch gives failures in the `staged-recipes (win_64 win)` job and `continuous-integration/appveyor/pr` jobs. Any suggestions to proceed?

---

Personally I think you can ignore that failure if nobody objects. It won't affect the feedstock as noarch packages are only built using CircleCI.

---

That's fine with me, thanks.